### PR TITLE
set up draft landing site

### PR DESCRIPTION
**Pull request by armandopmj**

### CODE REVIEW

File: db/schema.sql [hunk 1]
```diff
@@ -9,28 +9,23 @@ DROP TABLE IF EXISTS plans;
...
-    last_name VARCHAR(40) NOT NULL,
-    email VARCHAR(40) NOT NULL UNIQUE,
-    dob DATE NOT NULL,
+    first_name VARCHAR(40),
+    last_name VARCHAR(40),
+    email VARCHAR(40) UNIQUE,
+    dob DATE,
     phone BIGINT NOT NULL UNIQUE,
-    credit_card BIGINT UNIQUE,
+    registration_complete INT(1) NOT NULL,
```
Comment (mLuby) [inline db/schema.sql]: We could tell when registration is complete by checking if all the fields are not null. Just concerned we might end up with null fields and `registration_complete = true`, which would be very confusing.

File: server/api/users/users.main.js [hunk 2]
```diff
@@ -8,5 +8,6 @@ var router = express.Router();
 
 router.get('/', controller.getAll );
 router.post('/', controller.create );
+router.put('/:id', controller.finishRegistration );
 
 module.exports = router;
```
Comment (mLuby) [inline server/api/users/users.main.js]: new line

File: server/api/users/users.query.js [hunk 3]
```diff
@@ -8,6 +8,12 @@ exports.getAll = function( callback ) {
     });
 }
 
+exports.update = function( data, callback ) {
+    connection.query('UPDATE users SET ? WHERE user_id = ?;', data, function(err, results){
```
Comment (mLuby) [inline server/api/users/users.query.js]: Maybe security isn't a huge consideration just yet, but are we sanitizing the `data` that's being injected into SQL anywhere?

File: server/public/css/creative.css [hunk 4]
```diff
@@ -0,0 +1,543 @@
...
+    color: #fff;
+    background-color: #0ABAB5;
+    -webkit-transition: all .35s;
+    -moz-transition: all .35s;
+    transition: all .35s;
+}
+
+.btn-primary:hover,
+.btn-primary:focus,
+.btn-primary.focus,
```
Comment (mLuby) [inline server/public/css/creative.css]: Looks like some duplication in this file.

File: server/public/index.html [hunk 5]
```diff
@@ -0,0 +1,374 @@
+<!DOCTYPE html>
```
Comment (mLuby) [inline server/public/index.html]: stopping PR review here for now

File: server/public/index.html [hunk 6]
```diff
@@ -0,0 +1,374 @@
...
+    </header>
+    <!-- Splash Image End -->
+
+    <!-- What We Do -->
+    <section id="what">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center wow tada">
+                    <h1 class="section-heading">What we do</h1>
+                    <!-- <hr class="light"> -->
```
Comment (mLuby) [inline server/public/index.html]: Any reason you aren't deleting theses commented out portions?

File: server/public/js/libs/cbpAnimatedHeader.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+		didScroll = false;
+	}
+
+	function scrollY() {
+		return window.pageYOffset || docElem.scrollTop;
+	}
+
+	init();
+
+})();
```
Comment (mLuby) [inline server/public/js/libs/cbpAnimatedHeader.js]: the spacing in this and other files is inconsistent. Can we agree on something like 2 spaces, no tabs?

Also missing a final new line.

**Comment by mLuby** [comment]
@armandopmj looks good. I made some comments, but nothing important enough to stop you from merging whenever you like.

**Comment by armandopmj** [comment]
@mLuby thanks, I'll review and push any changes before a merge

**Pull request merged**